test/unit/StakingPool/depositTo.js

```diff
@@ -474,7 +474,7 @@ describe('depositTo', function () {
 
     const { firstActiveTrancheId, maxTranche } = await getTranches(daysToSeconds(0), DEFAULT_GRACE_PERIOD);
 
-    const tranches = Array(maxTranche - firstActiveTrancheId + 1)
+    const tranches = Array(maxTranche - firstActiveTrancheId)
```

Comment:
Not sure why this broke during this PR, but the StakingPool was reverting on this test when using 8 tranches. The js variable `firstActiveTrancheId` isn't the current firstActiveTrancheId in the contract, but is the first one available for this deposit, relative to `period` and `gracePeriod`. It was iterating past the max active tranches for this reason.